diff --git a/meson.build b/meson.build
--- a/meson.build
+++ b/meson.build
@@ -159,7 +159,7 @@
     'aes4000' : [ 'aeslib', 'aes3k' ],
     'uru4000' : [ 'nss' ],
     'elanspi' : [ 'udev' ],
-    'vfs0090' : [ 'nss', 'aes3k', 'openssl', 'submodule' ],
+    'vfs0090' : [ 'nss', 'aes3k', 'openssl' ],
     'virtual_image'          : [ 'virtual' ],
     'virtual_device'         : [ 'virtual' ],
     'virtual_device_storage' : [ 'virtual' ],
